client/src/app/components/managing-body-list/managing-body-list.component.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Created by amanecer on 16/05/2016.
+ */
+import {Component,Input,Output,EventEmitter} from '@angular/core';
+import {ManagingBodyComponent} from './managing-body/managing-body.component';
+
+
+@Component({
+    selector: 'op-managing-body-list',
+    templateUrl: 'app/components/managing-body-list/managing-body-list.component.html',
+    styleUrls: ['app/components/managing-body-list/managing-body-list.component.css'],
```

Comment:
We'll write SASS, please change the file extension.
Also, in PR #35 we'll add `Semantic-UI`, let's wait until we merge that PR and then do a re-roll and write your styles component.